### DISCUSSION TITLE
Blocked script execution in  because the document's frame is sandboxed and the 'allow-scripts' permission is not set.

### DISCUSSION THREAD

**Discussion by alanhe421** [coder/code-server#6208] [Q&A]
/stable-129500ee4c8ab7263461ffe327268ba56b9f210d/static/out/vs/workbench/contrib/webview/browser/pre/fake.html?id=2072224f-90d1-4771-8aee-3dd8fc2fd8fa:1 Blocked script execution in 'https://xxx/stable-129500ee4c8ab7263461ffe327268ba56b9f210d/static/out/vs/workbench/contrib/webview/browser/pre/fake.html?id=2072224f-90d1-4771-8aee-3dd8fc2fd8fa' because the document's frame is sandboxed and the 'allow-scripts' permission is not 

not work when  config locale and restart . and error occurs.

![image](https://github.com/coder/code-server/assets/9245110/f5e8c08c-3372-4326-8d72-ab57de707f60)

**Comment by code-asher** [comment]
Is this still not working in 4.13.0?  I think the blocked script execution is unrelated.

Restart should generate an `argv.json` file in your user data directory so one thing to do is check that it exists.

Also how was the extension installed?  Currently language bundles are only generated if they are installed through the UI so installing through the command line does not work unfortunately.
